### ISSUE TITLE
White page displayed when trying to install OWA

### ISSUE THREAD

**Issue by MickaelKicker** [padams/Open-Web-Analytics#295]
All needed permissions are granted, the requirements are fulfilled, but as soon as I enter the information for the config, the next page is nothing but a white page.
Not a single error message, not the slightest clue given as to what could be causing this problem.
Beyond the problem itself, the most annoying part is just how obscure an unhelpful the result is.

Using PHP7.1 with Nginx on Ubuntu Server 17.10

I pulled the master branch because the the release uses the deprecated mysql_connect(), which was removed from PHP7.

**Comment by padams** [comment]
What version of OWA and PHP?

Also turn on development logging in the config file to see what errors are thrown.

**Comment by SkipperHosting** [comment]
I noticed that one of my pages was able to give me clue as to what happened.  It appears that a few files have some whitespace after the closing php tag that muddles the output, causing this white screen (at least for me).

I removed the trailing whitespace and the closing php-tag altogether in the following files:
modules/base/entities/configuration.php
owa_entity.php
owa_env.php

There could be others, depending on which routines are hit.  It would suggest that all the php files be checked, or that the closing tag be removed to prevent such a situation
